app/services/table_service/create.rb

```diff
@@ -0,0 +1,23 @@
+# PATH: /app/services/table_service/create.rb
+# rubocop:disable Style/ClassAndModuleChildren
+class TableService::Create
+  attr_accessor :name, :table
+  def initialize(name)
+    @name = name
+  end
+  def call
```

Comment:
The `TableService::Create` service class is using manual validation which can be error-prone and hard to maintain. Consider using ActiveModel validations for better maintainability.